app/assets/stylesheets/application.scss

```diff
@@ -73,4 +73,24 @@ header {
 }
 .fa-icon {
   padding-right: 10px;
-}
\ No newline at end of file
+}
+
+//---FORM---//
+.form.form{
```

Comment:
this works?

Reply:
It does...it took a lot of inspecting to find the class to target this.